modules/appeals_api/app/services/appeals_api/slack/stuck_record_notification.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module Slack
+    class StuckRecordNotification
+      def initialize(records)
+        @records = records.sort_by { |r| r[:created_at] }
+      end
+
+      def message_text
+        <<~MESSAGE
+          :warning: The following appeal records are stuck:
+
+          #{records.map { |r| format_list_item(**r) }.join("\n")}
+        MESSAGE
+      end
+
+      private
+
+      attr_accessor :records
+
+      DATETIME_FORMAT = '%a %Y-%m-%d %H:%M:%S %Z'
```

Comment:
I think CMP prefers dates to be given to them in ISO-8601 format, which has a handy method via `created_at.iso8601`. That way we can just copy the records from the slack message to them & they'll have less to ask us to do :)